firedrake/__init__.py

```diff
@@ -1,19 +1,4 @@
 import firedrake_configuration
-import os
-import sys
-config = firedrake_configuration.get_config()
-if "PETSC_DIR" in os.environ and not config["options"]["honour_petsc_dir"]:
-    if os.environ["PETSC_DIR"] != os.path.join(sys.prefix, "src", "petsc")\
-       or os.environ["PETSC_ARCH"] != "default":
-        raise ImportError("PETSC_DIR is set, but you did not install with --honour-petsc-dir.\n"
-                          "Please unset PETSC_DIR (and PETSC_ARCH) before using Firedrake.")
-elif "PETSC_DIR" not in os.environ and config["options"]["honour_petsc_dir"]:
-    raise ImportError("Firedrake was installed with --honour-petsc-dir, but PETSC_DIR is not set.\n"
-                      "Please set PETSC_DIR (and PETSC_ARCH) before using Firedrake.")
-elif not config["options"]["honour_petsc_dir"]:  # Using our own PETSC.
-    os.environ["PETSC_DIR"] = os.path.join(sys.prefix, "src", "petsc")
-    os.environ["PETSC_ARCH"] = "default"
-del os, sys, config
```

Comment:
We don't use these variables anywhere. Also they only matter when we build petsc4py. At runtime this stuff is fixed.

Reply:
I see. I'll find a more graceful way to get Firedrake import working without removing this then.

Reply:
Sometimes `git annotate` provides useful context, in this case the relevant commit was 
fdccc0f7622568aac7391037cc32c88e34556ec8